pyproject.toml

```diff
@@ -5,5 +5,6 @@ description = "Add your description here"
 readme = "README.md"
 requires-python = ">=3.12"
 dependencies = [
+    "black>=25.1.0",
```

Comment:
I would group this as dev dependency, is there still a use for black where we have faster ruff?